### PR TITLE
fix: query builder groups on columns, not tag values

### LINKED ISSUE #16783: Group By in the query builder does not show column names
As a user
when i select group by in the visual data explorer
then i would like to see a list of relevant column names that I can select from

The current behavior seems to show the list of measurement values, which doesn't make sense in the context of the group by function. 

Looking at the signature for the group by function, it accepts a list of column names: https://v2.docs.influxdata.com/v2.0/reference/flux/stdlib/built-in/transformations/group/

The UI should show a list of relevant column names based on the query that has already been built.

### PR DESCRIPTION
**Pull request by hoorayimhelping**
Closes #16783 

A picture is worth a thousand words, so here:

**WRONG:**
Screen Shot 2020-02-24 at 2 43 26 PM

**Right:**
Screen Shot 2020-02-24 at 2 42 52 PM

* **Wrong**: The grouping options map to tag values
* **Right:** The grouping options map to columns

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [x] Tests pass

### CODE REVIEW

File: ui/src/timeMachine/selectors/index.ts [hunk 1]
```diff
@@ -244,25 +242,6 @@ export const getActiveTimeRange = (
...
-      tag.values.forEach(value => {
-        values.push(value)
-      })
-    })
-    return values
-  }
-
-  return activeQueryBuilderTags[index].values
-}
-
```
Comment (hoorayimhelping) [inline ui/src/timeMachine/selectors/index.ts]: this was testing the wrong thing.

File: ui/src/timeMachine/selectors/index.test.ts [hunk 2]
```diff
@@ -62,88 +58,3 @@ describe('TimeMachine.Selectors.Index', () => {
...
-
-  it('returns all previous tag values when the function is group', () => {
-    const actualTags = getActiveTagValues(activeQueryTags, 'group', 2)
-    expect(actualTags).toEqual([
-      ...activeQueryTags[0].values,
-      ...activeQueryTags[1].values,
-      ...activeQueryTags[2].values,
-    ])
-  })
-})
```
Comment (hoorayimhelping) [inline ui/src/timeMachine/selectors/index.test.ts]: this test was testing the wrong thing

File: ui/src/timeMachine/components/TagSelector.tsx [hunk 3]
```diff
@@ -49,6 +48,9 @@ import {
 
 const SEARCH_DEBOUNCE_MS = 500
 
+// We don't show these columns in results but they're able to be grouped on
```
Comment (121watts) [inline ui/src/timeMachine/components/TagSelector.tsx]: This is not always true.  There are scenarios where a `_start` and `_stop` are not not columns in a query response.

File: ui/src/timeMachine/selectors/index.test.ts (hunk 2 above)
Comment (121watts) [reply ui/src/timeMachine/selectors/index.test.ts]: can we test the right thing?

File: ui/src/timeMachine/components/TagSelector.tsx (hunk 3 above)
Comment (hoorayimhelping) [reply ui/src/timeMachine/components/TagSelector.tsx]: Russ mentioned that, but it seems like in the context of query builder, it's safe to assume we can group on those columns? I'll change the comment

File: ui/src/timeMachine/selectors/index.test.ts (hunk 2 above)
Comment (hoorayimhelping) [reply ui/src/timeMachine/selectors/index.test.ts]: >can we test the right thing?
@121watts only cause you asked https://github.com/influxdata/influxdb/pull/16992/files#diff-c432c510f4c140978dbe371ff3e9142dR93

**Pull request merged**